merlin-framework/src/main/java/gov/nasa/jpl/aerie/merlin/framework/ModelActions.java

```diff
@@ -12,18 +13,38 @@ protected ModelActions() {}
   static final Scoped<Context> context = Scoped.create();
 
 
+  public static <T> Context.TaskFactory threaded(final Supplier<T> task) {
+    return executor -> new ThreadedTask<>(executor, ModelActions.context, task);
+  }
```

Comment:
In order to spawn a lambda that has a return value, we'll also need to know how to serialize/deserialize that return value. It's not clear where that information comes from here.

Given the title of this PR, I suspect it would be okay to defer supporting lambdas-with-returns to a later pass on the topic?

Reply:
Only tasks that come from TaskSpecTypes will be deserialized (by virtue of having a "Directive"). So, tasks defined by lambdas are supported, (and in the future their return types can be made available to the caller), without any need for serialization.

Now, if it turns out we do want to include lambda tasks in serialization results... that's another problem entirely, I think.